ALUM/ALUM/Services/FirebaseAuthenticationService.swift

```diff
@@ -9,39 +9,40 @@ import SwiftUI
 import FirebaseAuth
 
 final class FirebaseAuthenticationService: ObservableObject {
-    var user: User? {
-        didSet {
-            objectWillChange.send()
-        }
+    static let shared = FirebaseAuthenticationService()
+
+    @ObservedObject var currentUser: CurrentUserModal = CurrentUserModal.shared
```

Comment:
This usage of one static object for `CurrentUserModel` is called the singleton pattern. Very useful for the use case of CurrentUserModel because we need to reuse this same information across the application. 


https://www.hackingwithswift.com/example-code/language/what-is-a-singleton